classes/Domain/Model/Talk.php

```diff
@@ -26,6 +26,9 @@
  * @method Builder ratedPlusOneBy(int $userId)
  * @method Builder viewedBy(int $userId)
  * @method Builder favoritedBy(int $userId)
+ *
+ * This is needed for Prophecy to mock the method, because it can't find it otherwise
+ * @method count()
```

Comment:
If https://github.com/phpspec/prophecy/pull/339 passes we can remove this, but for now, every function that isn't directly on the model itself has to be added like this.